src/user/proposals/ProposalEditor/EditorContent/EditorContent.js

```diff
@@ -428,4 +450,12 @@ class EditorContent extends Component {
   }
 }
 
-export default withRouter(EditorContent);
+const mapStateToProps = (state) => ({
+  createdProposal: state.createdProposal,
```

Comment:
Here is the issue @AuraOfDivinity 
change state.createdProposal to state.proposal.createdProposal  and
change state.fetchedProposal to state.proposal.fetchedProposal